src/foam/nanos/theme/ClientThemeService.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.theme',
+  name: 'ClientThemeService',
```

Comment:
Why is this needed rather than just letting FOAM generate the client along with all of the other services?

Reply:
I'm removing the service anyway, but I'll keep that in mind for the future.